sig-auth/charter.md

```diff
@@ -0,0 +1,68 @@
+# SIG Auth Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG Auth is responsible for the design, implementation, and maintenance of features in
+Kubernetes that control and protect access to the API and other core components. This includes
+authentication and authorization, but also encompasses features like auditing and some policy
+(see below).
+
+### In scope
+
+Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Kubernetes authentication, authorization, audit and policy features. Examples include:
```

Comment:
s/policy/security policy/? How does this overlap with other policy aspects?  Or do you think that it doesn't?

Reply:
Done. We actually were going to change that to security policy, but I guess that change got lost.

I agree this is a fuzzy area, and the policy scope of sig-auth isn't super well defined. I think that is partly because most policy features are cross-cutting. For example, PodSecruityPolicy should really be co-owned by sig-auth & sig-node. [SchedulingPolicy](https://github.com/kubernetes/community/pull/1937) should be sig-auth & sig-scheduling. Network policy is probably just sig-networking, since it's only a runtime policy (comparable to PodSecurityContext).

I see our (sig-auth) role around policy as:
- helping to drive consistency across policies (really co-owned with wg-policy)
- owning the policy primitives & mechanisms (co-owned with api-machinery)
- ensuring that important use cases are covered by the existing authorization & policy mechanisms

Does this make sense? I can try to distill this into our scope if we all agree.